invokeai/frontend/web/src/features/canvas/hooks/useCanvasZoom.ts

```diff
@@ -30,40 +33,57 @@ const useCanvasWheel = (stageRef: MutableRefObject<Konva.Stage | null>) => {
 
       e.evt.preventDefault();
 
-      const cursorPos = stageRef.current.getPointerPosition();
+      let delta = e.evt.deltaY;
```

Comment:
This variable isn't mutated when handling the brush size logic. Let's move it to be with the stage scale logic.

Reply:
Done